<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Sett</class>
 <widget class="QDialog" name="Sett">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>240</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(:/Images/BuffLogo3.png);</string>
  </property>
  <widget class="QPushButton" name="MemBtn">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>10</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/Images/mem.png</normaloff>:/Images/mem.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>52</width>
     <height>52</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="DateTimeBtn">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>160</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/Images/batt.png</normaloff>:/Images/batt.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>52</width>
     <height>52</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="UsrBtn">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>160</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/Images/add_user.png</normaloff>:/Images/add_user.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>52</width>
     <height>52</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="BackMenu">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>180</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/Images/back.png</normaloff>:/Images/back.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>40</width>
     <height>40</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="Storage">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>60</y>
     <width>59</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
   </property>
   <property name="text">
    <string>Storage</string>
   </property>
  </widget>
  <widget class="QLabel" name="Dt">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>210</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="User">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>210</y>
     <width>59</width>
     <height>16</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
   </property>
   <property name="text">
    <string>User</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
